kolibri/plugins/user_profile/assets/src/views/ChangeFacility/MergeUsernameExists.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
+    <p data-test="line1">
+      {{ mergeAccountInfoLine1 }}
+    </p>
+    <p data-test="line2">
+      {{ mergeAccountInfoLine2 }}
+    </p>
+
+    <BottomAppBar>
+      <slot name="buttons">
+        <KButtonGroup>
+          <KButton
+            :primary="false"
+            :text="coreString('backAction')"
+            appearance="flat-button"
+            data-test="backButton"
+            @click="sendBack"
+          />
+          <KButton
+            :primary="true"
+            :text="coreString('continueAction')"
+            data-test="continueButton"
+            @click="handleContinue"
+          />
+        </KButtonGroup>
+      </slot>
+    </BottomAppBar>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import { computed, inject } from 'kolibri.lib.vueCompositionApi';
+  import get from 'lodash/get';
+
+  export default {
+    name: 'MergeUsernameExists',
+    metaInfo() {
+      return {
+        title: this.$tr('documentTitle'),
+      };
+    },
+    components: { BottomAppBar },
+
+    mixins: [commonCoreStrings],
+
+    setup(_, context) {
+      const changeFacilityService = inject('changeFacilityService');
+      const state = inject('state');
+      const store =
+        context.root.$store !== undefined ? context.root.$store : context.root.$children[0].$store;
```

Comment:
This is non blocking, but I would probably move this logic into a well named function, replace the ternary operator with an if statement and assign it here instead. This would improve readability of this piece of code.